### ISSUE TITLE
A11y_mssql for VSCode_mssql Extension_Select top 1000 Results window_ColorContrast: Color contrast ratio between the keyboard focus and the background on the "save as json", "save as excel", "save as CSV" buttons is failing required ratio(3:1)

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-mssql#17612]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yTCS;#Win11;#DesktopApp;#BM_mssql for VSCode_Win32_Mar2023;#WCAG1.4.11;#A11ySev2;#Benchmark; #mssql for VSCode;#A11yMAS;#Win32;#AINotInScope;#A11yWCAG2.1;#ColorContrast;

## **Environment Details:**
Application Name: mssql for VSCode
Operating System: Win11[22H2]
OS Build: 22621.1413
Visual studio code insider: 1.77.0-insider (user setup)
Extension Version: v1.17.1

## **Repro Steps:**
1. Open Visual studio code
2. Tab till extension button and search for "mssql".
3. Tab till install and move to the SQL Server button.
4. Tab till Add Connection Button and insert the valid credentials.
5. Tab till the database and hit enter.
6. Tab till the System database and hit enter.
7. Tab till the Tables and hit enter.
8. Tab till any table under System tables and hit shift + F10
9. Tab till Select Top 1000 and hit enter 
10. Tab till Execute Query and hit enter 
11. Tab till "save as json"
12. Open Accessibility insights App on desktop and verify the color contrast ratio between the keyboard focus and the background on the "save as json", "save as excel", "save as CSV" buttons

## **Actual:**
Color contrast ratio between the keyboard focus and the background on the "save as json", "save as excel", "save as CSV" buttons is failing required ratio(3:1). Here the color contrast ratio is 2.397:1.

### Note: 
Issue is repro throughout the Application

## **Expected:**
Color contrast ratio between the keyboard focus and the background on the "save as json", "save as excel", "save as CSV" buttons should be greater than or equal to the required ratio 3:1. 

## **User Impact:**
Low vision users will face difficulty in figuring out the focus of the keyboard on save as json save as excel save as CSV buttons and think these buttons did not receive the keyboard focus.

## **Attachment**

![17612_A11y_mssql for VSCode_mssql Extension_Select top 1000 Results window_ColorContrast](https://user-images.githubusercontent.com/98821792/224705768-b9261d23-d3d3-4b8f-b01c-485ed541391c.png)

**Comment by alanrenmsft** [comment]
@prabhudevu please use the extension installer from here: https://microsoft-my.sharepoint.com/:u:/p/alanren/EQm7x3UTBmBMpsX5hOywRBAB0-RfVIeErCrSsxKJs8OBBg?e=e10YDN to verify the fix, you can follow the instructions here: https://code.visualstudio.com/docs/editor/extension-marketplace#_install-from-a-vsix

**Comment by prabhudevu** [comment]
Hi @alanrenmsft I have verified the issue in the below version after installing the provided extension

Version: 1.79.0-insider (user setup)
Commit: 2946b1ee55db1d406526d3fd1df49250b2f8322d
Date: 2023-05-18T05:25:02.319Z
Electron: 22.5.2
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621

Issue is fixed. Could you please close this issue as fixed.
![17612_Fixed](https://github.com/microsoft/vscode-mssql/assets/98821792/5ca49014-5ebb-4ea3-82ed-a625dfdad290)

**Comment by prabhudevu** [comment]
Closed this bug as fixed

**Comment by fsteffi** [comment]
#closed;
